finalPrep/annotations.js

```diff
@@ -3,40 +3,66 @@
 
 //Build a Bear
 function buildABear(name, age, fur, clothes, specialPower) {
+  //variable greeting assigned to the bear
   var greeting = `Hey partner! My name is ${name} - will you be my friend?!`;
+//variable demographics assigned to the parameters name and age
   var demographics = [name, age];
+// variable assigning a special power saying
   var powerSaying = "Did you know that I can " + specialPower + " ?";
+//variable assigned to build of the bear
   var builtBear = {
+//key value basic info
     basicInfo: demographics,
+//key value clothes
     clothes: clothes,
+//key value exterior
     exterior: fur,
+//key value cost
     cost: 49.99,
+//key value bear sayings
     sayings: [greeting, powerSaying, "Goodnight my friend!"],
+//key value cuddly
     isCuddly: true,
+//closing curly brace
   };
-
+//returns builtBear info
   return builtBear
+//closing curly brace
 }
-
+//instance of a bear
 buildABear('Fluffy', 4, 'brown', ['pants', 'jorts', 'tanktop'], 'give you nightmares');
+//instance of another bear
 buildABear('Sleepy', 2, 'purple', ['pajamas', 'sleeping cap'], 'sleeping in');
 
 
 
 //FizzBuzz
 function fizzBuzz(num1, num2, range) {
+//loop
   for (var i = 0; i <= range; i++) {
+//beginning of if/else if/else statement including conditionals
```

Comment:
What is the condition here? Explaining the condition is good practice in technical explanation. Also, you may need to research that operator `%` in the conditional.